src/schemas/avansertDokument/AvansertDokument.jsx

```diff
@@ -89,6 +89,20 @@ export default {
         },
       ],
     },
+    {
+      title: 'prioriteringsnummer',
+      type: SanityTyper.NUMBER,
+      name: DokumentNavn.PRIORITERINGSNUMMER,
+      description:
+        'Et nummer som brukes for å sortere brevmaler stigende i saksbehandlingsløsningen. Eksempel 1 vil vises først',
+      validation: Rule =>
+        Rule.custom(value => {
+          if (value < 1) {
+            return 'Tallet må være større enn eller lik 1';
+          }
+          return true;
+        }),
+    },
```

Comment:
Sanity har begynt å støtte litt mer typesikrign. Om vi importerer
```
import { Rule } from 'sanity';
```
Så kan vi skrive: 
```suggestion
      validation: (rule: Rule) =>
        rule.custom(value => {
          if (value < 1) {
            return 'Tallet må være større enn eller lik 1';
          }
          return true;
        }),
    },
```

Reply:
Går det an å "sortere" ting i Sanity basert på prioriteringsnummer også? Så saksbehandler ser de i stigende rekkefølge? 🤔

Reply:
I Sanity Studio er det også mulig å sortere på blant annet prioriteringsnummer. 
![image](https://github.com/navikt/familie-sanity-brev/assets/141132903/87ea1c81-4129-4893-8cd8-74876b7da301)

Reply:
Den der var ny for meg 🚀 Det går an å leke med deskstructure i sanity. Vi sorterer etter alfabetisk rekkefølge på tittel